app/presenters/monograph_presenter.rb

```diff
@@ -0,0 +1,3 @@
+class MonographPresenter < ::CurationConcerns::WorkShowPresenter
+  delegate :member_docs, :section_docs, to: :solr_document
```

Comment:
Why are we delegating to solr_doc instead of just adding that here?  Is there another scenario where it will be used?